app/assets/stylesheets/common/base/login.scss

```diff
@@ -50,6 +50,23 @@ body.invite-page {
   }
 }
 
+.password-reset-page {
+  .d-header .header-buttons .auth-buttons {
+    display: none;
+  }
+  .d-header-icons .search-dropdown {
+    display: none;
+  }
```

Comment:
Instead of hiding with CSS, which can be brittle should the classes change in the future. We can introduce a helper just like `hide-application-sidebar` and `hide-application-footer` to hide the buttons in the header.

Reply:
Okay will add a helper :+1: